feat: Emit event during PouchLifeCycle (#735)
cozy/cozy-client: docs/api/cozy-pouch-link.md

Normalizes the data (.id -> ._id, .rev -> _rev).
Passes the data to the client and to the onSync handler.

> Emit an event (pouchlink:sync:ended) when the sync (all doctypes) is done

Reviewer: ```suggestion
Emits an event (pouchlink:sync:end) when the sync (all doctypes) is done
```

**Kind**: instance method of [<code>PouchLink</code>](#PouchLink)  
<a name="PouchLink+startReplication"></a>

### pouchLink.startReplication() ⇒ <code>void</code>
User of the link can call this to start ongoing replications.
Typically, it can be used when the application regains focus.

> Emit pouchlink:sync:started event when the replication begins

Reviewer: ```suggestion
Emits pouchlink:sync:start event when the replication begins
```

User of the link can call this to stop ongoing replications.
Typically, it can be used when the applications loses focus.

> Emit pouchlink:sync:stoped event

Reviewer: ```suggestion
Emits pouchlink:sync:stop event
```

If you want to keep the "ed", it should be "stopped".
